src/AirtableRecord.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Yoanbernabeu\AirtableClientBundle;
+
+use DateTime;
+use DateTimeInterface;
+
+class AirtableRecord
+{
+    protected $fields;
+    protected string $id;
+    protected DateTimeInterface $createdTime;
+
+    public function __construct(string $id, $fields, DateTimeInterface $createdTime)
+    {
+        $this->fields = $fields;
+        $this->id = $id;
+        $this->createdTime = $createdTime;
+    }
+
+    public static function fromRecord(array $record): self
+    {
+        return new self($record['id'], $record['fields'], new DateTime($record['createdTime']));
```

Comment:
you should ensure that keys exist in $record otherwise you will have an error, you can use optionResolver but it's overkill so just check key with vanilla php 😛